src/components/MerchantPayout/MerchantPayoutScreen.tsx

```diff
@@ -201,16 +201,14 @@ export const MerchantPayoutScreen: FunctionComponent<NavigationFocusInjectedProp
 
   return (
     <>
-      <ScrollView
-        contentContainerStyle={{ alignItems: "center" }}
-        scrollIndicatorInsets={{ right: 1 }}
-        keyboardShouldPersistTaps="handled"
-      >
-        <TopBackground mode={config.appMode} />
-        <KeyboardAvoidingView
-          behavior={Platform.select({ ios: "position" })}
-          keyboardVerticalOffset={Platform.select({ ios: -80 })}
+      <KeyboardAvoidingView behavior={Platform.select({ ios: "padding" })}>
```

Comment:
Seems like this is always needed as a wrapper around `ScrollView`, so was wondering if we can refactor this out into a component for reuse?

Reply:
Good point! Previously the `KeyboardAvoidingView` was always scattered as a descendent of some `ScrollView`. Looks like it makes sense and works well when the `KeyboardAvoidingView` wraps the `ScrollView` instead.